README.md

```diff
@@ -1,23 +1,64 @@
-# Socket Runtime
+
+<p align="center">
+  <a href="https://github.com/socketsupply/socket"><img src="https://user-images.githubusercontent.com/79177582/219756869-afe51bf1-e047-4fab-9e4f-654c5bfd5393.jpeg" width="10%" height="10%"/>
+</p>
+<h1 align="center">Socket Runtime</h1>
+                                             
+<h2 align="center">⚡Write once, run anywhere, connect everyone⚡</h2>  
+
+
+  
+  
 
 ### Description
 
-Socket is a client-side runtime for creating native cross-platform software on
-mobile and desktop using HTML, CSS, and JavaScript. It also exposes primitives
-needed for building peer-to-peer and local-first applications, such as Bluetooth,
-UDP and File I/O.
+Web Developers use `Socket Runtime` to create apps for any OS, desktop, or mobile. You can use plain old HTML, CSS, and JavaScript, as well as your favorite front-end libraries for example React, Svelt, and Vue.
+
+`Socket Runtime` exposes primitives needed for building peer-to-peer and local-first applications, such as Bluetooth, UDP, and robust file system access. Our P2P component can help you connect your app's users, and let them communicate directly, without the cloud or any servers at all.
+
+The Socket's CLI compiles applications into hybrid-native applications &mbasp; meaning, a combination of web code running in a platform's "WebView" along with platform-native code: Java on Android, Swift/Objective-C on iOS, C++ on Windows or Linux, etc.
```

Comment:
```suggestion
The Socket's CLI compiles applications into hybrid-native applications &mbasp; meaning, a combination of web code running in a platform's "WebView" along with platform-native code: Kotlin/Java on Android, Swift/Objective-C on iOS, C++ on Windows or Linux, etc.
```

Reply:
@chicoxyzzy for the description/intro was suggested by @heapwolf so I did the update, I'll update the readme with your changes.

Reply:
> The Socket's CLI
should be `The Socket Runtime CLI`

Reply:
I will make the change now